README.md

```diff
@@ -1,20 +1,28 @@
 1. Replace `<your_account>` with your Github username in the link
     - [DEMO LINK](https://<your_account>.github.io/js_notification_DOM/)
 2. Follow [this instructions](https://mate-academy.github.io/layout_task-guideline/)
-    - There are no tests for this task so use `npm run lint` command instead of `npm test` 
+    - Run `npm run test` command to test your code;
+    - Run `npm run test:only -- -n` to run fast test ignoring linter;
+    - Run `npm run test:only -- -l` to run fast test with additional info in console ignoring linter.
 
 ### Task: Create a notification
 
 To inform the user about the success of the operation, or any errors, websites usually use messages that appear for a while on the screen.
 
-Your task: Create function `pushNotification`, which takes coords of future message, title, description and type (warning, success or error), appends it to the document and remove it after 2 seconds.
-You no need to write any styles or HTML code in this task. Change only `main.js` file.
+Your task: Create function `pushNotification`, which takes coordinates of the message, title, description, and type 
+(success, error and warning). `pushNotification` creates an element to display the message, appends it to the document, and removes it from the DOM after 2 seconds.
+Do not write any CSS styles or HTML code in this task. You should modify only `main.js` file.
 
 Implementation tips:
-- The message is block element with class `notification` + class which depends on the input parameter type (`success`, `error`, `warning`). 
-- The message should have a title with class `title` (prefer `h2` element)
-- The message should have description (prefer tag `p`)
-- Set coordinates of the message (top and right) rely on values from parameters.
+- Print three messages: `success`, `error`, `warning` (see image below);
```

Comment:
explain that function calls are already in main.js

Reply:
done